landmarks.json

```
@@ -1,4 +1,443 @@
+    },
+    {
+        "wikidata": "Q1809404",
+        "note": "Laykyun Sekkya",
+        "maki_temaki": "maki/monument"
+    },
+    {
+        "wikidata": "Q113070852",
+        "note": "Statue of Belief",
+        "maki_temaki": "maki/monument"
+    },
+    {
+        "wikidata": "Q542324",
+        "note": "Ushiku Daibutsu",
+        "maki_temaki": "maki/monument"
+    },
+    {
+        "wikidata": "Q3118755",
+        "note": "Guishan Guanyin",
+        "maki_temaki": "maki/monument"
+    },
+    {
+        "wikidata": "Q2083114",
+        "note": "Great Buddha of Thailand",
+        "maki_temaki": "maki/monument"
+    },
+    {
+        "wikidata": "Q1621882",
+        "note": "Sendai Daikannon",
+        "maki_temaki": "maki/monument"
+    },
+    {
+        "wikidata": "Q28153889",
+        "note": "Dai Kannon",
+        "maki_temaki": "maki/monument"
+    },
+    {
+        "wikidata": "Q243",
+        "note": "Eiffel Tower",
+        "maki_temaki": "maki/monument"
+    },
+    {
+        "wikidata": "Q4152",
+        "note": "Schloss Neuschwanstein",
+        "maki_temaki": "maki/castle"
+    },
+    {
+        "wikidata": "Q1413217",
+        "note": "Žižkov Television Tower",
+        "maki_temaki": "temaki/tower"
+    },
+    {
+        "wikidata": "Q212065",
+        "note": "Edinburgh Castle",
+        "maki_temaki": "maki/castle"
+    },
+    {
+        "wikidata": "Q47476",
+        "note": "Alhambra",
+        "maki_temaki": "maki/castle"
+    },
+    {
+        "wikidata": "Q183536",
+        "note": "Tokyo Tower",
+        "maki_temaki": "maki/monument"
+    },
+    {
+        "wikidata": "Q223207",
+        "note": "Oriental Pearl Tower",
+        "maki_temaki": "temaki/tower"
+    },
+    {
+        "wikidata": "Q756268",
+        "note": "Stirling Castle",
+        "maki_temaki": "maki/castle"
+    },
+    {
+        "wikidata": "Q9141",
+        "note": "Taj Mahal",
+        "maki_temaki": "maki/place-of-worship"
+    },
+    {
+        "wikidata": "Q12506",
+        "note": "Hagia Sophia",
+        "maki_temaki": "maki/place-of-worship"
+    },
+    {
+        "wikidata": "Q69513",
+        "note": "Pena Palace",
+        "maki_temaki": "maki/castle"
+    },
+    {
+        "wikidata": "Q129846",
+        "note": "St. Basil's Cathedral",
+        "maki_temaki": "temaki/domed_tower"
+    },
+    {
+        "wikidata": "Q151356",
+        "note": "Fernsehturm Berlin",
+        "maki_temaki": "temaki/tower"
+    },
+    {
+        "wikidata": "Q48435",
+        "note": "Sagrada Família",
+        "maki_temaki": "maki/place-of-worship"
+    },
+    {
+        "wikidata": "Q178114",
+        "note": "Washington Monument",
+        "maki_temaki": "temaki/obelisk"
+    },
+    {
+        "wikidata": "Q390150",
+        "note": "Salisbury Cathedral",
+        "maki_temaki": "temaki/obelisk"
+    },
+    {
+        "wikidata": "Q901058",
+        "note": "Vehicle Assembly Building",
+        "maki_temaki": "maki/castle-JP"
```

```suggestion
        "maki_temaki": "temaki/hangar"
```
While not quite the right shape, i think it's a more logical choice
https://github.com/rapideditor/temaki/blob/main/icons/hangar.svg

---

Yeah I was between those two. Went for the building outline rather than intended purpose. I could be persuaded

---

I don't think the shape is that similar, and it's strangely simplistic compared to all the other icons.

---

It probably also contributed to my not being able to guess what it was from the name and icon (re including NASA above)